src/controllers/dashboard/dashboard.js

```diff
@@ -175,7 +175,8 @@ import confirm from '../../components/confirm/confirm';
 
             if (!result.Items.length) {
                 view.querySelector('.activeRecordingsSection').classList.add('hide');
-                return void(itemsContainer.innerHTML = '');
+                itemsContainer.innerHTML = '';
+                return undefined;
```

Comment:
There is no need to explicitly return `undefined` in current browsers. Just `return;` should be fine.

Reply:
Got it, just thought you wanted me to add `undefined` explicitly. Thanks.